tests/data_model/test_generated_result.py

```diff
@@ -1,50 +0,0 @@
-# Copyright The Caikit Authors
```

Comment:
tests are now here: https://github.com/caikit/caikit/blob/main/tests/interfaces/nlp/test_text_generation.py with the use of caikit.interfaces data model